### PR TITLE
Miscellaneous Documentation Updates

### PR DESCRIPTION
**Pull request by kkeirstead**
Mostly fixing dead links and small typos.

### CODE REVIEW

File: documentation/collectionrules.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Actions may reference the outputs of other actions that have started before them
 
 where `<ActionName>` is the name of the action from which to get an output value and `<OutputName>` is the name of an output from that action.
 
-For example, if action `A` has an output named `EgressPath`, and action `B` has a settings property named `Arguments`, then action `B` can reference the `EgressPath` from within the `Arguments` property setting:
+For example, if action `A` has an output named `Egress`, and action `B` has a settings property named `Arguments`, then action `B` can reference the `Egress` from within the `Arguments` property setting:
```
Comment (jander-msft) [inline documentation/collectionrules.md:78]: Please revert; the output name is `EgressPath`: https://github.com/dotnet/dotnet-monitor/blob/main/src/Tools/dotnet-monitor/CollectionRules/Actions/CollectionRuleActionConstants.cs#L9

File: documentation/setup.md [hunk 2]
```diff
@@ -57,4 +57,4 @@ The official releases and previews are:
 - Documented on the [Releases](./releases.md) page.
 - Announced on the GitHub [Releases](https://github.com/dotnet/dotnet-monitor/releases) tab.
 - Published to [nuget.org](https://www.nuget.org/packages/dotnet-monitor/).
-- Published to the Microsoft Container Registry ([tag list](https://mcr.microsoft.com/v2/dotnet/monitor/tags/list)).
\ No newline at end of file
+- Published to the Microsoft Container Registry ([tag list](https://mcr.microsoft.com/v2/dotnet/monitor/tags/list)).
```
Comment (jander-msft) [inline documentation/setup.md:60]: What changed here?

File: documentation/troubleshooting.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
 # Troubleshooting Guide
 
-Here is our guide for diagnosing specific issues with `Dotnet-Monitor`. This is an ongoing effort to add issues and solutions to this guide and is never complete but you can help us by letting us know in the issues sections if we should add something here
+Here is our guide for diagnosing specific issues with `dotnet monitor`. This is an ongoing effort to add issues and solutions to this guide; you can help us by letting us know in the Issues section if we should add something here.
```
Comment (jander-msft) [inline documentation/troubleshooting.md:3]: Should "Issues section" link to https://github.com/dotnet/dotnet-monitor/issues?

File: documentation/troubleshooting.md [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
+If you find that `dotnet monitor` is consuming lots of CPU time, this typically means that there is a high load of logging statements being emitted from the process. `dotnet monitor` is used to capture `AppServiceAppLogs` and `AllMetrics` from your process before sending them onto other systems; these settings are configured under Monitoring > Diagnostic Settings in the Azure Portal. `dotnet monitor` is also always enabled to collect a baseline level of metrics from the process to display in the Azure Portal. However, metrics monitoring is fast enough that it should not show up as an impact on CPU time.
```
Comment (jander-msft) [inline documentation/troubleshooting.md:7]: "fast enough" -> "lightweight such" or "low impact such"

The metrics are not fast, they are low impact.

File: documentation/setup.md (hunk 2 above)
Comment (kkeirstead) [reply documentation/setup.md:60]: It may have added a newline; there should be no change here.

**Review by jander-msft — APPROVED** [review]

**Pull request merged**